README.md

```diff
@@ -56,665 +56,15 @@ To run Bio-OS, the following prerequisites must be met:
-
-Local environment testing can be quickly completed with the following commands.
-```shell
-# 进入开源 bioos 目录,执行命令
-docker-compose up -d
-```
-**Note**: Local deployment will involve the problem of jupyterhub dynamically obtaining tokens. You can refer to the online deployment part. To update the token, you need to restart the bioos-apiserver container.
-
-2. Online Deployment
-
-Online deployment requires the preparation of a container orchestration engine, and we provide helm deployment packages to quickly complete bioos deployment.
-
-- Deployment based on Kubernetes environment
-
-If there is no local Kubernetes environment, you can try to deploy minikube. For details, please refer to the [official website](https://minikube.sigs.k8s.io/docs/start/).
-
-When installing the container runtime, you have two options:
-- Docker - The  most common container runtime environment
-  - Deployment method
-  ```shell
-  sudo apt update && apt install  docker.io -y
-  ```
-- Nerdctl - Open source and open operating environment, a perfect alternative to the cncf community
-  - Deployment method
-  ```shell
-    wget https://github.com/containerd/nerdctl/releases/download/v1.4.0/nerdctl-full-1.4.0-linux-amd64.tar.gz
-    tar zxf nerdctl-full-1.4.0-linux-amd64.tar.gz -C /usr/local/
-    cp /usr/local/lib/systemd/system/*.service /etc/systemd/system/
-    ln -s /usr/local/bin/nerdctl /usr/bin/docker
-    systemctl enable buildkit containerd
-    systemctl restart buildkit containerd
-  ```
-Kubernetes environment also has many deployment methods, common deployment forms such as minikube/microk8s/kubeadm/kubespray, considering the simplicity and ease of use, here take minikube as an example to build the local environment, production environment recommends choosing kubespray for high availability deployment.
-```shell
-# 安装基础依赖
-sudo apt update && apt install -y conntrack
-sudo sysctl fs.protected_regular=0 #重启之后，记得执行一下，避免 minikube /tmp/juju-mkf6a06118463380f4d96c12aced04598f450743: permission denied 类似报错
-mkdir -p /etc/containerd && containerd config default >  /etc/containerd/config.toml
-wget https://github.com/kubernetes-sigs/cri-tools/releases/download/v1.27.0/crictl-v1.27.0-linux-amd64.tar.gz
-tar xvf crictl-v1.27.0-linux-amd64.tar.gz -C /usr/bin/
-# 安装 minikube
-curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
-sudo install minikube-linux-amd64 /usr/local/bin/minikube
-# 使用国内源安装 kubernetes
-minikube start \
---container-runtime="containerd" \
---image-mirror-country=cn \
---image-repository=registry.cn-hangzhou.aliyuncs.com/google_containers \
---driver=none \
---apiserver-ips='xxx.xxx.xx.xx' # 可选配置，如果需要外网访问，需要这里配置主机的外网IP
-```
-Deployment log information:
-```shell
-root@registry:/home/vagrant# minikube start --container-runtime="containerd" --image-mirror-country=cn --driver=none --image-repository=registry.cn-hangzhou.aliyuncs.com/google_containeres
```

Comment:
seperate executable command line and stdout/stderr

Reply:
The file is not maintained in this PR, so I will ignore it.